src/Mono.Android/Android.Runtime/AndroidEnvironment.cs

```diff
@@ -254,8 +294,12 @@ static string GetDefaultTimeZone ()
 		static SynchronizationContext GetDefaultSyncContext ()
 		{
 			var looper = Android.OS.Looper.MainLooper;
+			try {
 			if (Android.OS.Looper.MyLooper() == looper)
```

Comment:
I think we're going to want to reindent this/similar code so that it actually looks "correct".  We'll just have to ignore whitespace changes while reviewing this PR so that it doesn't look ridiculously huge.